packages/opentelemetry-node-tracer/src/instrumentation/PluginLoader.ts

```diff
@@ -90,8 +90,6 @@ export class PluginLoader {
           `PluginLoader#load: trying loading ${name}.${version}`
         );
 
-        // @todo (issues/132): Check if version and supportedVersions are
-        // satisfied
         if (!version) return exports;
```

Comment:
I let this check because we can fail fast (without doing require(moduleName))... if it's not a big issue, I can remove this check.